### PR TITLE
Updated rows of table to split hosted vs cloud

### PR DESCRIPTION
**Pull request by IanLee1521**
Splits the *.com / *.org from the self-hosted repo hosting services. For instance, Scraper currently supports Bitbucket Server (self-hosted) but not Bitbucket.org due to differences in their APIs.

### CODE REVIEW

File: CODE_JSON_GENERATORS.md [hunk 1]
```diff
@@ -26,12 +29,45 @@ From the working group session we have captured business needs,  system requirem
 
 Please see the full list here: [https://github.com/GSA/code-gov/projects/1](https://github.com/GSA/code-gov/projects/1)
 
+## GSA Code Inventory Tool
+
+The [Code Inventory](https://github.com/GSA/codeinventory) tool is a Ruby gem application that provides basic functionality in a CLI and API, and it can be extended via plug-ins (which are also gems) to support any type of repository
+One plug-in we developed is to add GitHub support: [https://github.com/GSA/codeinventory-github
+]()
+
+[https://github.com/GSA/code-clerk](New Code Clerk tool)
```
Comment (froi) [inline CODE_JSON_GENERATORS.md]: Wrong link formatting. It should be `[New Code Clerk tool](https://github.com/GSA/code-clerk)`

File: CODE_JSON_GENERATORS.md [hunk 2]
```diff
@@ -26,12 +29,45 @@ From the working group session we have captured business needs,  system requirem
 
 Please see the full list here: [https://github.com/GSA/code-gov/projects/1](https://github.com/GSA/code-gov/projects/1)
 
+## GSA Code Inventory Tool
+
+The [Code Inventory](https://github.com/GSA/codeinventory) tool is a Ruby gem application that provides basic functionality in a CLI and API, and it can be extended via plug-ins (which are also gems) to support any type of repository
+One plug-in we developed is to add GitHub support: [https://github.com/GSA/codeinventory-github
```
Comment (froi) [inline CODE_JSON_GENERATORS.md]: This seems weird. If I preview the doc this link doesn't seem to work: `[https://github.com/GSA/codeinventory-github]()`

**Comment by froi** [comment]
@IanLee1521 Let me know if what I'm pointing out was not introduced by you. Thanks!

**Comment by froi** [comment]
Thanks for the fixes @IanLee1521!

**Comment by IanLee1521** [comment]
Thank YOU!

**Pull request merged**